src/core/pd_repo.js

```diff
@@ -0,0 +1,83 @@
+import mongoose from "mongoose";
+
+import data_chunks from "../constants/pd_constants.js";
+
+import logger from "./pd_logger.js";
+
+class repo {
```

Comment:
rename class to schemaRepository same for file

Reply:
done.